Neos.Flow/Tests/Functional/ObjectManagement/Fixtures/PHP81/BackedEnumWithMethod.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace Neos\Flow\Tests\Functional\ObjectManagement\Fixtures\PHP81;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations\Proxy;
+
+/**
+ * A PHP 8.1 value-backed enum with methods
+ */
+enum BackedEnumWithMethod: string
```

Comment:
I assume that this fixture is not loaded with PHP < 8.1 and doesn't cause any harm that way!?

Reply:
Yes, exactly what you wondered about in [CompileTimeObjectManager.php](https://github.com/neos/flow-development-collection/pull/2702/files/6519b87dd9b6dbab2ee5b8c352ab108ffc53917d#diff-e4637b28695cb70185ef6ac4a5b949b4051de8e9844f2c5f797b472ea66884aa)